.github/ISSUE_TEMPLATE/regression_bug_report.md

```diff
@@ -0,0 +1,65 @@
+---
+name: Regression Impact Report
+labels: type:Testing
```

Comment:
Somewhat tangential to this, but we need to have a *concise* description of the intended use of labels.  (summarize https://discourse.itk.org/t/github-issue-labels/1256/11)

< ! - - Labels have 3 categories that can be assigned "area: (blue hued label), type (yellow hued label):, status: (purple hued label)",  Please consider picking one label from each of these categories in order to add easily searchable informtion to the issue. -->

My critique of how the labeling system could be improved:

critique:  I think the "type:Bug" issue is too ambiguous in that is really overlaps with all other types.  in some respects all the other type: issues are all bugs.   

critique:  A new label category would complete the statndard sorting of importantce for "ugency:[0Critial | 1High | 2Medium | 3Low | 4Unnecessary]".  Color from bright red to white in decending color scheme.

critique:  All labels need a description.

critique: status labels should have a progression "status: [0Active | 0Blocked | 2Confirmed | 3NeedsInfo | 4Backlog]" in gradient of dark purple to light purple.

Reply:
> critique: A new label category would complete the statndard sorting of importantce for "ugency:[0Critial | 1High | 2Medium | 3Low | 4Unnecessary]". Color from bright red to white in decending color scheme.

I disagree -- importance is subjective. To the reporter, it is almost often **Critical**, in their eyes. Other community members may have different priorities for an issue.  Also, we already have enough label categories ...

Reply:
All current labels **have a description now**.

As for the **importance or urgency and status** labels, I do not have a strong opinion, even if it was me that brought the issue to the table 😊 . I think that they can be useful at times, but I'm not sure whether people are likely to embrace them due to the overhead they bring (even to the maintainers -e.g. an issue is marked as critical, but it turns out it is not), or their power may be somehow lost if used only sparingly.

We may add a notice in the templates to tell users to be gentle when marking the importance/urgency if this bring us to a meeting point.

As for the colors, although I think the current code is visual enough, the 3-color code suggestions seems fair.

Again, mine is not a strong opinion, so I will embrace whatever you the heavy ITK'ers decide.

Reply:
The urgency was a suggestion based on using labels for project management.  Don't worry about them now, it is not a priority for ITK at the moment.